src/payments/payments-types.ts

```diff
@@ -0,0 +1,425 @@
+export namespace Payments {
+    export interface ErrorResponse {
+        error: {
+            /** @enum {string} */
+            type: "INTERNAL" | "AUTHENTICATION" | "AUTHORIZATION" | "VALIDATION" | "NOT_FOUND" | "UNPROCESSABLE_ENTITY" | "FORBIDDEN";
+            message: string;
+        };
+    }
+
+    export type CorrelationData = AnyAdditionalPropertiesSchema | string;
+
+    export interface AnyAdditionalPropertiesSchema {
+        [key: string]: unknown;
+    }
```

Comment:
that's fine.
since we need to allow `Record<string, unknown>` values, in the `openapi` the way to do that is by a schema of an object with `additionalProperties: { }`.
This is the reason for this weird interface.
We can change here every usage of this interface to the actual type, but in my opinion we can leave it like that for now.